src/mlpack/methods/reinforcement_learning/q_networks/simple_dqn.hpp

```diff
@@ -63,21 +61,21 @@ class SimpleDQN
       network(outputLayer, init),
       isNoisy(isNoisy)
   {
-    network.Add(new Linear<>(inputDim, h1));
-    network.Add(new ReLULayer<>());
+    network.Add(new mlpack::ann::Linear<>(inputDim, h1));
```

Comment:
Not sure I can see why this was necessary, it doesn't improve the readability.

Reply:
I am removing only `mlpack::`, but we have to keep the ann since we can not use namspaces in header file